agenta-cli/agenta/cli/variant_commands.py

```diff
@@ -97,14 +97,13 @@ def add_variant(
 
     # update the config file with the variant names from the backend
     variant_name = f"{base_name}.{config_name}"
-    overwrite = False
 
     client = AgentaApi(
         base_url=f"{host}/{BACKEND_URL_SUFFIX}",
         api_key=api_key,
     )
 
-    if variant_name in config["variants"]:
+    if variant_name in config["variants"] and not overwrite:
```

Comment:
this behavior is a bit weird. So if overwrite is set to False, you ask the question?

Reply:
I would rename it to force_overwrite or skip_overwrite_question

Reply:
@mmabrouk 

it's a cli flag. 
so if you pass it you say explicitly to overwrite the variant.
if you don't you get a confirmation question

```
~/b/a/agenta/examples/baby_name_generator optional-interactive-mode ?1 ❯ agenta variant serve --file_name app.py --overwrite                                               ✘ INT  agenta-py3.12

Checking and updating config file...
Preparing code base app into a tar file...
Building code base app for app.default into a docker image...
Updating app to server...
Variant app.default for App woddooo updated successfully 🎉

~/b/a/agenta/examples/baby_name_generator optional-interactive-mode ?1 ❯ agenta variant serve --file_name app.py                                                              8s  agenta-py3.12

Checking and updating config file...
? This variant already exists. Do you want to overwrite it? (Y/n)                                                                                                              
```

Reply:
I understand that, I just mean in that code, the function logic does not make sense. I would have called the argument in the function `force_overwrite` or `skip_overwrite_question` because otherwise the logic does not make sense (if not overwrite, then ask the use whether to overwrite. The correct logic would be if not skip_overwrite_question then ask the user whether to overwrite). But I see you've already merged..